modules/programs/neovim.nix

```diff
@@ -172,7 +172,7 @@ in {
       extraLuaPackages = mkOption {
         type = with types; either extraLua51PackageType (listOf package);
         default = [ ];
-        defaultText = "[]";
+        defaultText = literalExpression "[ ]";
```

Comment:
Could simply remove this, the `default = [ ]` should render just fine in the documentation.